notebooks/dataAccess/siphon-RemoteAccess.ipynb

```diff
@@ -0,0 +1,408 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<a name=\"top\"></a>\n",
+    "<div style=\"width:1000 px\">\n",
+    "\n",
+    "<div style=\"float:right; width:98 px; height:98px;\">\n",
+    "<img src=\"https://raw.githubusercontent.com/Unidata/MetPy/master/src/metpy/plots/_static/unidata_150x150.png\" alt=\"Unidata Logo\" style=\"height: 98px;\">\n",
+    "</div>\n",
+    "\n",
+    "<h1>Siphon (remote_access)</h1>\n",
+    "<h3>Unidata AMS 2021 Student Conference</h3>\n",
+    "\n",
+    "<div style=\"clear:both\"></div>\n",
+    "</div>\n",
+    "\n",
+    "---\n",
+    "\n",
+    "In this notebook, we'll covering opening, inspecting, subsetting, and plotting a TDS dataset using Siphon's `remote_access` method.\n",
+    "<div style=\"float:right; width:250 px\"><img src=\"../../instructors/images/siphon_remote_access_preview.png\" alt=\"plot of data accessed via Siphon's remote_access\" style=\"height: 300px;\"></div>\n",
+    "\n",
+    "\n",
+    "### Focuses\n",
+    "* Use Siphon `remote_access` to open a TDS dataset\n",
+    "* Access dataset using both [CDM Remote](https://www.unidata.ucar.edu/software/netcdf-java/v4.5/reference/stream/CdmRemote.html) and [OPENDAP](https://www.opendap.org/)\n",
+    "* Subset and download variables in dataset\n",
+    "* Plot downloaded data\n",
+    "\n",
+    "\n",
+    "### Objectives\n",
+    "1. [Find a dataset in a TDS Catalog](#1.-Find-a-dataset-in-a-TDS-Catalog)\n",
+    "1. [Access the dataset using `remote_access`](#2.-Access-the-dataset-using-remote_access)\n",
+    "1. [Use the remote dataset to subset, download, and display data](#3.-Use-the-remote-dataset-to-subset,-download,-and-display-data)\n",
+    "\n",
+    "---"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Before beginning, let's import the packages to be used throughout this training:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import matplotlib.pyplot as plt\n",
+    "import numpy as np\n",
+    "from siphon.catalog import TDSCatalog"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "---"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 1. Find a dataset in a TDS Catalog\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Before we use `remote_access`, we need to find a dataset that we'd like to access.  \n",
+    "As an example, we'll use this [dataset](https://thredds-test.unidata.ucar.edu/thredds/catalog/grib/nexrad/composite/unidata/NEXRAD_Unidata_Reflectivity-20201207/Level3_Composite_N0R_20201207_0000.grib2/catalog.html?dataset=grib/nexrad/composite/unidata/NEXRAD_Unidata_Reflectivity-20201207/Level3_Composite_N0R_20201207_0000.grib2) from in the Unidata THREDDS test catalog."
```

Comment:
This is what these two links showed for me (see attached photo).  I'm not sure if that is what's to be expected or not?
![Screen Shot 2021-01-07 at 4 15 58 PM](https://user-images.githubusercontent.com/21114460/103955508-a7638100-5103-11eb-886e-73df51ab6f4d.png)

Also, this should be reworded:
> **from in** the Unidata THREDDS test catalog.